src/Command/InfectionCommand.php

```diff
@@ -325,7 +326,7 @@ private function includeUserBootstrap(InfectionConfig $config): void
 
         if ($bootstrap) {
             if (!file_exists($bootstrap)) {
-                throw LocatorException::fileOrDirectoryDoesNotExist($bootstrap);
+                throw FileNotFound::createForFile($bootstrap);
```

Comment:
I did not realise that exception was used elsewhere this way, that said in this specific case it feels a `LogicException` (instead of a `RuntimeException` which is what `FileNotFound` is) is more appropriate since it is a configuration error that requires immediate fix from the user